lib/ios_analytics/translate.rb

```diff
@@ -3,9 +3,36 @@
 
 module IosAnalytics
   module Translate
-    def trans(checkstyle_dom, plist_file)
+    def trans(derivedData, appName)
```

Comment:
__[rubocop]__
* Method has too many lines. [20/10] [:ref](https://github.com/bbatsov/ruby-style-guide#short-methods)
* Use snake_case for variable names. [:ref](https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)
* Use snake_case for variable names. [:ref](https://github.com/bbatsov/ruby-style-guide#snake-case-symbols-methods-vars)